testapps/test_pillowtop/tests/test_settings.py

```diff
@@ -1,12 +1,13 @@
 import os
 from django.conf import settings
-from django.test import TestCase
+from django.test import TestCase, override_settings
 import json
 from corehq.util.test_utils import TestFileMixin
 from pillowtop.listener import AliasedElasticPillow, BasicPillow
 from pillowtop.utils import get_all_pillow_configs
 
 
+@override_settings(DEBUG=True)
```

Comment:
yeah, i actually added it at first yesterday and decided this was simpler. happy to add it back if you think that's better